### ISSUE TITLE
Error on 'react-native link react-native-sentry'

### ISSUE THREAD

**Issue by SirNeuman** [getsentry/react-native-sentry#284]
OS:
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

_Platform:_
- [x]  iOS
- [x]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
v8.3.0
5.3.0
WallPixi@1.0.0 /Users/joshuasacks/Pixilated/WallPixi/Pixilated
├── babel-plugin-transform-remove-console@6.8.5
├── base-64@0.1.0
├── lodash@4.17.4
├── moment@2.19.1
├── moment-timezone@0.5.13
├── prop-types@15.6.0
├── react@16.0.0-beta.5
├── react-native@0.49.3
├── react-native-background-timer@1.2.0
├── react-native-camera@0.10.0
├── react-native-config@0.7.1
├── react-native-countdown-circle@1.0.2 
├── react-native-fetch-blob@0.10.8
├── react-native-google-signin@0.12.0
├── react-native-keep-awake@2.0.5
├── react-native-network-info@3.0.0
├── react-native-orientation@3.1.0
├── react-native-restart-android@0.0.6
├── react-native-sentry@0.29.0
├── react-native-sha256@1.1.1
├── react-native-ssdp@2.8.2
├── react-native-udp@2.1.0 
├── react-native-vector-icons@4.4.0
├── react-native-view-shot@2.1.0
├── react-redux@5.0.6
├── redux@3.7.2
├── redux-saga@0.15.6
├── redux-thunk@2.2.0
└── xmldom@0.1.27
```

---
I have following issue:

On running `react-native link react-native-sentry` after running `npm install react-native-sentry --save` I get an error. This is the output of my shell after running the command:

```
rnpm-install info Linking react-native-sentry android dependency
rnpm-install info Android module react-native-sentry has been successfully linked
rnpm-install info Linking react-native-sentry ios dependency
rnpm-install WARN ERRGROUP Group 'Frameworks' does not exist in your Xcode project. We have created it automatically for you.
rnpm-install ERR! Something went wrong while linking. Error: Cannot read property 'children' of null
Please file an issue here: https://github.com/facebook/react-native/issues

Cannot read property 'children' of null
```

It appears to have updated some of my Android files (android/settings.gradle, MainApplication.java, and android/app/build.gradle) but none of my iOS files and I was never given any prompts for DNS stuff like the documentation says it should ask for after running the link step.

**Comment by HazAT** [comment]
Hey, I am not able to reproduce this:
If I run:
```bash
react-native init AwesomeProject
cd AwesomeProject
yarn add react-native-sentry
react-native link react-native-sentry
```

I get this:
```bash
rnpm-install info Linking react-native-sentry android dependency
rnpm-install info Android module react-native-sentry has been successfully linked
rnpm-install info Linking react-native-sentry ios dependency
rnpm-install WARN ERRGROUP Group 'Frameworks' does not exist in your Xcode project. We have created it automatically for you.
rnpm-install info iOS module react-native-sentry has been successfully linked
```

**Comment by SirNeuman** [comment]
Hmmm... yeah I wasn't able to reproduce it either with a clean React-Native project and all of my package dependencies. I'm gonna keep trying to find out where the issue is within my specific configuration in my project. If I find it I'll post it here. Even if it's not an issue with react-native-sentry it could help anyone else who happens to run into this issue.

**Comment by SirNeuman** [comment]
Ok. So I've tried copying over parts of my original project into a clean project one by one and trying the linking step to see when i get that error. It appears the issue is within the `ios` directory of my react-native project. Still not sure why thats happening though. I haven't been working with ios at all (been focusing solely on Android), and seeing as how i've been using the react-native-git-upgrade with my RN project for a few months now it might be kind of hard for me to pin down where this error is. I'm wondering if I can/should just do a clean rebuild of the ios portion of my codebase.

**Comment by SirNeuman** [comment]
So I found a couple of other places in other libraries where people had a similar problem with react-native linking for iOS: https://github.com/BranchMetrics/react-native-branch-deep-linking/issues/239 and https://github.com/alwx/react-native-http-bridge/issues/1. I also found that i ran into the same issue when i tried to do `react-native unlink react-native-google-signin`. But didn't have the issue when I tried to unlink any other project. 

So I'm not sure what these libraries have in common in terms of their linking step, but for me the best solution is probably just to regenerate the iOS portion of my app since I haven't been working with it anyway. I found this to do that: https://stackoverflow.com/questions/42506068/how-can-i-regenerate-ios-folder-in-react-native-project and then I did `react-native link react-native-sentry` and I did not get the error
